addons/website_forum/views/forum_forum_templates.xml

```diff
@@ -208,9 +208,11 @@
                     <input type="hidden" name="karma_edit_retag" t-att-value="forum.karma_edit_retag" id="karma_edit_retag"/>
                     <t t-set="edit_tags_karma_fail" t-value="user.karma &lt; forum.karma_edit_retag"/>
                     <t t-set="edit_tags_karma_error_message">You need to have sufficient karma to edit tags</t>
-                    <input type="text" name="post_tags" class="form-control js_select2" placeholder="Tags" t-attf-data-init-value="#{tags}" value="Tags"
+                    <!-- TODO: add condition for readonly & title -->
+                    <!-- <input type="text" name="post_tags" class="form-control js_select_menu" placeholder="Tags" t-attf-data-init-value="#{tags}" value="Tags"
                         t-att-readonly="edit_tags_karma_fail and 'readonly'"
-                        t-att-title="edit_tags_karma_fail and edit_tags_karma_error_message"/>
+                        t-att-title="edit_tags_karma_fail and edit_tags_karma_error_message"/> -->
```

Comment:
?

Reply:
working on it. will update it soon. Thanks : )

Reply:
Update: Hello @mcm-odoo, Done with the TODO.